packages/titanium-data-table/src/titanium-data-table.ts

```diff
@@ -332,6 +332,16 @@ export class TitaniumDataTableElement extends LitElement {
       fill: var(--app-light-text-color, #80868b);
     }
 
+    sub-actions {
+      display: flex;
+      min-width: 100%;
+      justify-content: flex-end;
+    }
+
+    sub-actions > ::slotted(*) {
+      margin: 8px 0 8px 16px;
```

Comment:
Why not even padding?

Reply:
I wanted it to bump up to the edge on the right side.  I could make it 8px all around, but then you will see 8px off from the edge.